book/modules/appendix2-exercises.tex

```diff
@@ -1,9 +1,301 @@
+				\[
+					x\vec v_1+y\vec v_2+z\vec v_3=\vec 0.
+				\]
+				
+				The vector equation is equivalent to the system of linear equations
+				\[
+					\systeme{-4x-2y-2z=0,8y+7z=0,x+3z=0,7x+3y+4z=0}.
+				\]
+				
+				The solution to this system is
+				\[
+					\mat{x\\y\\z}=\mat{0\\0\\0}.
+				\]
+				
+				Since the only solution is the trivial solution, the vectors $\vec v_1$, $\vec v_2$ and $\vec v_3$ are linearly independent.
+				\item The lines $\ell_1$ and $\ell_2$ intersect when the components of $\vec x$ are equal. Equating the components of $\vec x$ gives the equations
+				\[
+					\systeme{t+1=2s+3,3t+1=s+4}.
+				\]
+				
+				The solution to this system is
+				\[
+					\mat{t\\s}=\mat{4/5\\-3/5}.
+				\]
+				
+				Since $\vec x=\mat{9/5\\17/5}$ when $t=4/5$, the intersection of $\ell_1$ and $\ell_2$ is the point $\mat{9/5\\17/5}$.
+				\item 
+				Since $\vec v_1, \vec v_2, \vec v_3\in\R^3$, they span $\R^3$ if they are linearly independent. We need to determine if there is a non-trivial solution to
+				\[
+					x\vec v_1+y\vec v_2+z\vec v_3=\vec 0.
+				\]
+				
+				This is equivalent to the system of linear equations
+				\[
+					\systeme{x-2y+2z=0,2x+y+7z=0,3x+z=0}.
+				\]
+				
+				The solution to this system is
+				\[
+				\mat{x\\y\\z}=\mat{0\\0\\0}.
+				\]
+				
+				Since the only solution is the trivial solution, the vectors $\vec v_1$, $\vec v_2$ and $\vec v_3$ are linearly independent.
+			\end{enumerate}
+		\end{solution}
+		\prob Presented below are other students' work for the previous problem. Evaluate whether their reasoning correctly answer the question, and point out the flaws in their reasoning, if any.
+		\begin{enumerate}
+			\item 
+			The following answer is for part (a):
+			
+			The vectors $\vec v_1$, $\vec v_2$ and $\vec v_3$ are linearly independent when there does not exist a non-trivial solution to
+			\[
+				x\vec v_1+y\vec v_2+z\vec v_3=\vec 0.
+			\]
+			\item 
+			The following answer is for part (b):
+			
+			The lines $\ell_1$ and $\ell_2$ intersect when the components of $\vec x$ are equal. Equating the components of $\vec x$ gives the equations
+			\[
+				\systeme{t+1=2t+3,3t+1=t+4}.
+			\]
+			
+			This system is equivalent to
+			\[
+				\systeme{t=-2,2t=3}.
+			\]
+			
+			Since this system is inconsistent, there does not exist a solution to this system, so the lines $\ell_1$ and $\ell_2$ do not intersect.
+			\item 
+			The following answer is for part (b):
+			
+			The lines $\ell_1$ and $\ell_2$ intersect when the components of $\vec x$ are equal. Equating the components of $\vec x$ gives the equations
+			\[
+				\systeme{t+1=2s+3,3t+1=s+4}.
+			\]
+			
+			This system is equivalent to
+			\[
+				\systeme[ts]{t-2s=2,3t-s=3}.
+			\]
+			
+			The solution to this system is
+			\[
+				\mat{t\\s}=\mat{4/5\\-3/5},
+			\]
+			so the lines $\ell_1$ and $\ell_2$ intersect at $\mat{4/5\\3/5}$.
+		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item 
```

Comment:
Blank item?

Reply:
Part (a) is not finished since I was not able to come up with some common errors for that, so I left this one blank.